### ISSUE TITLE
Exception when upgrading to latest React.NET version

### ISSUE THREAD

**Issue by meisterpeeps** [reactjs/React.NET#484]
I tried updating to 3.2 and I am now getting a `React.Exceptions.ReactNotInitialisedException: 'ReactJS.NET has not been initialised correctly.'`

That includes three exceptions:
-`TinyIoCResolutionException: Unable to resolve type: React.ReactEnvironment`
-`TinyIoCResolutionException: Unable to resolve type: React.JavaScriptEngineFactory`
-`ReactNotInitialisedException: React has not been loaded correctly. Please expose your version of React as global variables named 'React', 'ReactDOM' and 'ReactDOMServer', or enable the 'LoadReact' configuration option to use the built-in version of React.`

Other than upgrading the nuget packages associated with React.NET and moving to React 16.0.0  from 15.4.1,  I have changed nothing else.   Before the update all was well.    
Any thoughts? :/  Thanks for any help.

**Comment by dustinsoftware** [comment]
A few things to check:

- Does the error occur if you update to React 16 without updating React.NET? It works, but will throw a warning at runtime saying the server-render call is deprecated.
- Try disabling server-render and see if your component renders on the page. You can do so by passing `clientOnly: true` as an argument to the render call in your razor file

**Comment by Daniel15** [comment]
It sounds like you're calling `.SetLoadReact(false)` but aren't properly loading React yourself. Try instead using ReactJS.NET's built-in React version (by removing the `.SetLoadReact(false)` call).

**Comment by meisterpeeps** [comment]
Just want to mention that I wasn't able to get 3.2 working, but I was able to get 3.1 working--as I was updating from 3.0.  As for your two suggestions; I tried the clientOnly: true, and the problem didnt go away; and as far I know I have been loading React myself correctly, as it worked on 3.0 and works on 3.1.  
Anyway, thanks for the quick feedback.  If I try again to go for 3.2, and if I get any new error information, or it just happens to work, Ill let you know.
